### ISSUE TITLE
Does dygraph branch support plato2's interact mode?

### ISSUE THREAD

**Issue by zhanghaoie** [PaddlePaddle/Knover#107]

**Comment by sserdoubleh** [comment]
It can't support plato2's interact mode now. We will upgrade dygraph branch later.

**Comment by zhanghaoie** [comment]
thank you very much for your reply! Is it possible to convert the pretrained model saved with fluid.io.save_params to .pdparams format?

**Comment by sserdoubleh** [comment]
> thank you very much for your reply! Is it possible to convert the pretrained model saved with fluid.io.save_params to .pdparams format?

You can use `fluid.io.load_program_state`. It loads parameters in numpy format, and you need to convert them to .pdparams format by changing their name.

**Comment by zhanghaoie** [comment]
ok, thank you very much for your reply

**Comment by zhanghaoie** [comment]
Hi，is it possible to load .pdparams model for fine-tuning with develop branch?

**Comment by sserdoubleh** [comment]
> Hi，is it possible to load .pdparams model for fine-tuning with develop branch?

Develop branch cannot support .pdparams now. Do you need to convert other pretrained model into Knover?

**Comment by zhanghaoie** [comment]
yeah, i have found paddlenlp releases a plato-mini model and want to use it for fine-tuning.

**Comment by sserdoubleh** [comment]
OK, I will update some convert tool later.

**Comment by zhanghaoie** [comment]
> OK, I will update some convert tool later.

Hi, how about the convert tool.
